app/layout.tsx

```diff
@@ -1,31 +1,41 @@
-import Header from '@/components/Header'
-import Providers from '@/providers/Providers'
-import React from 'react'
-import '@/styles/global.css'
-import Image from 'next/image'
+"use client";
```

Comment:
Why does `layout` need to be made to `use client`???
- How could you add `use client` to one of the children and remove it from `layout`?